Updated intakes.md (#223)
SEKOIA-IO/documentation: docs/operation_center/intakes.md

# Intakes - Data Sources Management

> Intakes correspond to data sources sent to SEKOIA.IO. They are identified by a name, an entity and an 'intake key', which is used to configure the sending from your information system.  

Reviewer: ```suggestion
Intakes correspond to data sources sent to SEKOIA.IO. They are identified by a name, an entity and an “intake key”, which is used to configure the sending from your information system.
```

> All features related to intakes are visible through the `Intakes` menu on the Operation Center.

Reviewer: ```suggestion
All features related to intakes are visible through the “Intakes” menu on the Operations Center.
```

The intakes homepage will allow you to view the list of intakes already created within the community but also to create intakes.

> ## Intakes list

Reviewer: ```suggestion
## Intake List
```

> On the intakes list, the number of events sent to SEKOIA.IO for the past 7 days and the number of events in error (not integrated) are displayed, as well as the intake key, with a button to copy this value to the clipboard.

Reviewer: ```suggestion
On the intake list, the number of events sent to SEKOIA.IO for the past 7 days and the number of events in error (not integrated) are displayed, as well as the intake key, with a button to copy this value to the clipboard.
```

![image](https://user-images.githubusercontent.com/94441479/158421171-a0f7a1a8-4c9b-4c9f-acae-97a00f6e0fc9.png)

## Create an intake using the integrations catalog

> After clicking on the dedicated button `+ Intake`, you will be able to create an intake with an existing integration developped by SEKOIA.IO's team.

Reviewer: ```suggestion
After clicking on the dedicated button “+ Intake”, you will be able to create an intake with an existing integration developed by SEKOIA.IO team.
```

> You just need to choose the right source in the catalog and click on `Create`. 

Reviewer: ```suggestion
You just need to choose the right source in the catalog and click on “Create”.
```

> Here is the information you should provide to validate the creation :

Reviewer: ```suggestion
Here is the information you should provide to validate the creation:
```

- An intelligible name 
- An entity among the list to which you want to associate the corresponding data
- The format of the events

![image](https://user-images.githubusercontent.com/94441479/158421961-15b69898-a806-4d67-af0e-e52e2e359fc4.png)

> You will then be able to find your intake in the list as well as the associated `intake key` which will allow you to set up different elements such as symphony modules.

Reviewer: ```suggestion
You will then be able to find your intake in the list as well as the associated intake key which will allow you to set up different elements such as Symphony modules.
```
